test/e2e/client_tls_test.go

```diff
@@ -1670,3 +1615,57 @@ func curlGetStatusCode(t *testing.T, clientPod *corev1.Pod, certName, endpoint,
 		return httpStatusCodeInt, nil
 	}
 }
+
+func buildCRLHostPod(podName, namespaceName, crlConfigMapName string) corev1.Pod {
+	return corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      podName,
+			Namespace: namespaceName,
+			Labels:    map[string]string{"app": podName},
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{{
+				Name:  "httpd",
+				Image: "quay.io/openshifttest/httpd-24:multi",
+				Ports: []corev1.ContainerPort{{
+					ContainerPort: 8080,
+					Name:          "http-svc",
+				}},
+				VolumeMounts: []corev1.VolumeMount{{
+					Name:      "data",
+					MountPath: "/var/www/html",
+					ReadOnly:  true,
+				}},
+				SecurityContext: generateUnprivilegedSecurityContext(),
+			}},
```

Comment:
This is subtle.  You factored this function out of `TestCRLUpdate`, but you also changed the image pullspec from `quay.io/centos7/httpd-24-centos7:centos7` to `quay.io/openshifttest/httpd-24:multi` and removed the readiness probe.  Were those changes intentional?  If they were, can you use separate commits, one adding `buildCRLHostPod` and one changing image pullspec and probe?  It also helps if the commit message explains the changes.

Reply:
Good catch; that wasn't intentional. I had to do some manual merging when I rebased recently, so either I missed that difference, or it got dropped by git. Either way, I'll restore those changes.

Reply:
Should be fixed. I've re-added the readiness probe and changed it to use `quay.io/centos7/httpd-24-centos7:centos7`.